homeassistant/components/overkiz/number.py

```diff
@@ -225,3 +229,14 @@ async def async_set_native_value(self, value: float) -> None:
         await self.executor.async_execute_command(
             self.entity_description.command, value
         )
+
+    async def async_added_to_hass(self) -> None:
```

Comment:
This should be done in the `__init__` function? See https://github.com/home-assistant/core/blob/aa1d970b7ccb3f3ab8597ab38dd533b9c74bd4df/homeassistant/components/overkiz/light.py#L42.